doc/user/content/get-started/install-materialize-emulator.md

```diff
@@ -106,6 +106,23 @@ not suitable for full feature set evaluations or production workloads.
 For questions, discussions, or general technical support, join the [Materialize
 Community on Slack](https://materialize.com/s/chat).
 
+#### `mz-debug`
```

Comment:
As an fyi -- we'll need to manually add this to https://github.com/MaterializeInc/materialize/blob/main/doc/user/content/integrations/_index.md page as well.